static/src/javascripts-legacy/projects/common/modules/email/run-checks.js

```diff
@@ -30,6 +30,7 @@ define([
     Promise
 ) {
     var emailInserted = false;
+    var compliantOutbrain = false;
```

Comment:
If compliantOutbrain is a property of outbrain, then run-checks would require outbrain, and outbrain would require run-checks.  Are circular dependencies allowed?